apps/studio/src/pages/home/uploadSong/BasicSongDetails.tsx

```diff
@@ -401,10 +404,19 @@ const BasicSongDetails: FunctionComponent<BasicDonDetailsProps> = ({
                     "ownership, makes streaming royalties available for " +
                     "purchase, and enables royalty distribution to your account."
                   }
-                  disabled={ isDeclined }
+                  disabled={
+                    isDeclined || webStudioDisableTrackDistributionAndMinting
+                  }
                   includeBorder={ false }
                   name="isMinting"
                   title="DISTRIBUTE & MINT SONG"
+                  toggleTooltipText={
+                    webStudioDisableTrackDistributionAndMinting
+                      ? "Track distribution and minting is temporarily disabled. " +
+                        "Please upload your song to save your progress, and check " +
+                        "back later to finish the distribution process."
+                      : undefined
+                  }
```

Comment:
This shows up in the `Library` edit song page when the minting is already toggled by default, I'm able to even proceed to next, logic needs to be added to disable this route as well.

![image](https://github.com/user-attachments/assets/eec1e35b-8728-4e7f-9fc2-b73bc190a77a)